manifests/profile/hathitrust/apache.pp

```diff
@@ -8,13 +8,150 @@
 #
 # @example
 #   include nebula::profile::hathitrust::apache
-class nebula::profile::hathitrust::apache () {
-  package { [
-      'apache2',
-# default?
-#      'apache2-mpm-prefork',
-# removed, will use fcgid or proxy?
-#      'libapache2-mod-fastcgi',
-    ]:
+class nebula::profile::hathitrust::apache (
+  String $prefix = '',
+  String $domain = 'hathitrust.org',
+  String $sdrroot = '/htapps/babel',
+) {
+
+  $default_access = {
+    enforce  => 'all',
+    requires => [
+      'not env badrobot',
+      'not env loadbalancer',
+      'all granted'
+    ]
+  }
+
+  $haproxy_ips = nodes_for_class('nebula::profile::haproxy').map |String $nodename| {
+    fact_for($nodename, 'networking')['ip']
+  }
+
+
+  class { 'apache':
+    default_vhost          => false,
+    default_ssl_vhost      => false,
+    # changed from default 300 (copypasta)
+    timeout                => 900,
+    # changed from default 15 (puppet), 5 (debian) (copypasta)
+    keepalive_timeout      => 2,
+    log_formats            => {
+      combined => '%a %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\" %v \"%{X-HathiTrust-InCopyright}o\"'
+    },
+    # configured below by explicitly declaring params for apache::mod::prefork class
+    mpm_module             => false,
+    serveradmin            => 'lit-ae-systems@umich.edu',
+    servername             => $domain,
+    trace_enable           => 'Off',
+    root_directory_secured => true,
+    scriptalias            => undef,
+    docroot                => false,
+    default_mods           => false,
+    user                   => 'nobody',
+    group                  => 'nogroup',
+  }
+
+  class { 'apache::mod::prefork':
+    # changed from default; copypasta from debian 8 config
+    startservers           => 10,
+    minspareservers        => 10,
+    maxspareservers        => 15,
+    maxrequestworkers      => 256,
+    maxconnectionsperchild => 0
+  }
+
+  # VERIFY no settings for <Directory /usr/share>
+  # VERIFY no settings for <Directory /var/www>>
```

Comment:
Verified for both `/usr/share` and `/var/www` there are no `Directory` settings for them on the reified config on ht-web-001, so we can remove these comments.